.vsts/darwin/distribution.yml

```diff
@@ -14,6 +14,23 @@ steps:
     displayName: Build packages
 
   - script: |
+      set -e
+      dir=$(Agent.TempDirectory)
+      keychain=$dir/buildagent.keychain
+      security create-keychain -p pwd $keychain
+      security default-keychain -s $keychain
+      security unlock-keychain -p pwd $keychain
+      echo "$(apple-developer-certificate)" | base64 -D > $dir/cert.p12
+      security import $dir/cert.p12 -k $keychain -P "$(apple-developer-certificate-key)" -T /usr/bin/codesign
+      security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k pwd $keychain
+
+      echo "##vso[task.setvariable variable=CSC_LINK]$dir/cert.p12
+      echo "##vso[task.setvariable variable=CSC_KEY_PASSWORD]$(apple-developer-certificate-key)
```

Comment:
It still seems odd that we're having to do code-signing at the Electron Builder level then again via ESRP. Just to double check, if you remove this script then the *.plist files aren't copied into the final package?

Reply:
Those steps seem necessary, pending clarification from the ESRP team on how we can use the ADO task to add the entitlements.